cli/ingress-controller/flags.go

```diff
@@ -83,6 +83,11 @@ type cliConfig struct {
 	APIServerHost      string
 	KubeConfigFilePath string
 
+	// Allowed Ingress resource versions
+	AllowIngressExtensionsV1beta1 bool
+	AllowIngressNetworkingV1beta1 bool
+	AllowIngressNetworkingV1      bool
```

Comment:
Why are these configuration options being added?
We want the controller to automatically figure out the resource version to use and then communicate with the API-server accordingly.

Reply:
Discussed out of band.
@mflendrich Let us make this opt out instead. This means these configuration options should be of the opposite polarity and default by false.

Reply:
Done in dd7e691

I'm not a fan of flags with "negative semantics" (that is, I prefer `--enable-X=true` instead of `--disable-X=false` by default) because the latter usually come with an extra `!` somewhere in the code (here, in `main.go`). But it's probably not worth bikeshedding in this instance.